src/openforms/emails/templates/emails/templatetags/form_steps_summary.html

```diff
@@ -20,7 +20,8 @@ <h3 style="margin: 0 0 1em; font-size: 1.2em;">{{ submission_step_node.render }}
             {% with add_bottom_margin=forloop.last %}
                 {% if component_node.label or component_node.display_value %}
                     {% ifchanged component_node.layout_modifier %}
-                        {% if component_node.layout_modifier == 'fieldset' %}
+                        {% has_bold_label component_node.layout_modifier as make_bold%}
+                        {% if make_bold %}
```

Comment:
This could/should be done as providing the modifiers in the rendering context rather than inventing a custom template tag for this. I feel the custom tag  is too specific and spreads out the formatting logic across different tags/template files.